core/src/main/java/org/opensearch/sql/storage/StorageEngine.java

```diff
@@ -18,14 +19,14 @@ public interface StorageEngine {
   /**
    * Get {@link Table} from storage engine.
    */
-  Table getTable(String name);
+  Table getTable(CatalogSchemaName catalogSchemaName, String tableName);
```

Comment:
why not https://github.com/opensearch-project/sql/blob/2.x/core/src/main/java/org/opensearch/sql/analysis/model/CatalogSchemaIdentifierName.java?

Reply:
CatalogSchemaIdentifier is both for catalogSchemaTable name and catalogSchemaFunction name. So created a new model for catalog schema.

Reply:
Does CatalogSchemaIdentifier.getCatatlog() and CatalogSchemaIdentifier.getSchema achieve same thing as CatalogSchemaName done in here. 
Try to clear the pros/cons between
* Table getTable(CatalogSchemaName catalogSchemaName, String tableName);
* Table getTable(CatalogSchemaIdentifier tableIdentifier);

Reply:
Not much difference actually Since CatalogSchemaIdentifier is also having resolver logic, I thought of creating a clean separate POJO for catalogschema. I can change the interface.

Reply:
I perfer CatalogSchemaIdentifier, and CatalogSchemaName is not necessary then. Thanks.